dataminer-overview/Support Services/Support_services_agreement.md

```diff
@@ -45,7 +45,7 @@ Skyline provides Support Services for its Software only, which consists of its c
 
 Support and Updates for the Software shall only be provided on the two latest main releases of the Software. A new main release is issued by Skyline about every year. Main releases contain all new features and cumulative updates that were issued since the previous main release.
 
-It is the User's responsibility to make sure that their Software is updated timely to a supported Software version. An up-to-date Support timeline shall be available on [DataMiner Dojo](https://community.dataminer.services/dataminer-supported-versions-lifecycle/).
+It is the User's responsibility to make sure that their Software is updated timely to a supported Software version. An up-to-date Support timeline shall be available on [docs.dataminer.services](xref:Skyline_support_life_cycles).
```

Comment:
```suggestion
It is the user's responsibility to make sure that their Software is updated timely to a supported Software version. An up-to-date Support timeline shall be available on [docs.dataminer.services](xref:Skyline_support_life_cycles).
```

Reply:
This is capitalized because this page contains a legal agreement. So while it looks a bit odd, it's for a reason :)